### PR TITLE
[AF-2310]: Added infinispan ping service to prevent spawning threads

### PR DESCRIPTION
**Pull request by adrielparedes**
## Related to: 
* https://github.com/kiegroup/appformer/pull/838
* https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/1092
* https://github.com/kiegroup/kie-wb-common/pull/2991
* https://github.com/kiegroup/jbpm-wb/pull/1401
* https://github.com/kiegroup/drools-wb/pull/1256
* https://github.com/kiegroup/kie-wb-distributions/pull/986
* https://github.com/kiegroup/optaplanner-wb/pull/349

### CODE REVIEW

**Comment by adrielparedes** [comment]
jenkins execute full downstream build

**Comment by adrielparedes** [comment]
jenkins execute full downjenkins execute full downstream build

**Comment by adrielparedes** [comment]
jenkins execute full downstream build

**Comment by adrielparedes** [comment]
jenkins execute full downstream build

**Comment by adrielparedes** [comment]
jenkins execute full downstream build

File: uberfire-extensions/uberfire-metadata/uberfire-metadata-commons-io/src/main/java/org/uberfire/ext/metadata/io/BatchIndex.java [hunk 1]
```diff
@@ -78,6 +78,12 @@ public BatchIndex(final MetaIndexEngine indexEngine,
     }
 
     public void runAsync(final FileSystem fs) {
+
+        if(!this.indexEngine.isAlive()){
```
Comment (tomasdavidorg) [inline uberfire-extensions/uberfire-metadata/uberfire-metadata-commons-io/src/main/java/org/uberfire/ext/metadata/io/BatchIndex.java]:
```suggestion
        if (!this.indexEngine.isAlive()) {
```

**Pull request merged**